ElfPack/asm/W595_R3EG004_Orange.asm

```diff
@@ -16,12 +16,10 @@ a	EQU	b
 	defadr	 DB_PATCH5_RET,0x1185F87C+1
 	defadr	 DB_PATCH6_RET,0x1185F8B0+1
 	defadr	 MESS_HOOK_RET,0x1127B5D0+1
-	defadr	 KEY_HOOK_REPEAT_RETUN,0x10A33938+1
-	defadr	 KEY_HOOK_TIMER_RETUN,0x10A3399E+1

```

Comment:
there are still unused KEY_HOOK_REPEAT_RETUN and KEY_HOOK_TIMER_RETUN in C510_R1FA035_Orange.asm